### PR TITLE
Create new blog post about WebPositive

### PR DESCRIPTION
**Pull request by jt15s**
Added new blog post on behalf of animortis, talking about Web+

### CODE REVIEW

File: content/blog/animortis/2021-08-10_positively_critical_webpositive_and_haiku.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+The history of WebPositive is as intricately woven into Haiku as BeOS was, and has served as a major effort for all the programmers and users of the operating system even as other web browsers are successfully ported, work for a little while, then, unfortunately, lose functionality and are abandoned.  
+
+Developer PulkoMandy, whose real name is Adrien Destugues, lives in France and has been a key developer for Haiku and WebPositive since joining the team in 2009.  
+
+WebPositive, sometimes called “Web+,” uses the WebKit1 engine. This engine was originally ported from KDE’s KHTML by Apple for their Safari browser and later brought into Google’s Chrome, Samsung’s phone browsers and more.  
+
+“Few other browser engines have such portability and utility,” PulkoMandy said.  
+
+Haiku developer Ryan Leavengood started efforts to port the engine to create WebPositive in 2008. In 2009, WebPositive won a slot as part of the Google Summer of Code, and student developer Maxine Simon was able to get the browser in a “mostly usable” state while developer stippi built the browser framework itself.
```
Comment (leavengood) [inline content/blog/animortis/2021-08-10_positively_critical_webpositive_and_haiku.md:21]:
```suggestion
Haiku developer Ryan Leavengood started efforts to port the engine to create WebPositive in 2008. In 2009, WebPositive won a slot as part of the Google Summer of Code, and student developer Maxime Simon was able to get the browser in a “mostly usable” state while developer Stephan Aßmus (stippi) built the browser framework itself.   
```

**Comment by kallisti5** [comment]
@pulkomandy should really review this one :-)

File: content/blog/animortis/2021-08-10_positively_critical_webpositive_and_haiku.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+WebPositive, sometimes called “Web+,” uses the WebKit1 engine. This engine was originally ported from KDE’s KHTML by Apple for their Safari browser and later brought into Google’s Chrome, Samsung’s phone browsers and more.  
+
+“Few other browser engines have such portability and utility,” PulkoMandy said.  
+
+Haiku developer Ryan Leavengood started efforts to port the engine to create WebPositive in 2008. In 2009, WebPositive won a slot as part of the Google Summer of Code, and student developer Maxine Simon was able to get the browser in a “mostly usable” state while developer stippi built the browser framework itself.   
+
+When PulkoMandy was hired, first as a paid developer before switching over as a volunteer to work on the WebKit port under WebPositive, there wasn’t a ton of new development going on with the software.  
+
+WebPositive wasn’t the only browser for Haiku. But with the additional portability and power of the WebKit1 engine as it existed at the time, the team decided to build it into  WebPositive and make it the official browser for the OS.
```
Comment (kallisti5) [inline content/blog/animortis/2021-08-10_positively_critical_webpositive_and_haiku.md:25]: grammar. Starting a sentence with "But"

**Review by leavengood — CHANGES_REQUESTED** [review]
Very nice article. I just wanted to correct the spelling of Maxime's name, and also decided to add stippi's full name, though later there is a big list of nicknames,

File: content/blog/animortis/2021-08-10_positively_critical_webpositive_and_haiku.md (hunk 2 above)
Comment (jt15s) [reply content/blog/animortis/2021-08-10_positively_critical_webpositive_and_haiku.md]: @kallisti5 Fixed!

File: content/blog/animortis/2021-08-10_positively_critical_webpositive_and_haiku.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+WebPositive, sometimes called “Web+,” uses the WebKit1 engine. This engine was originally ported from KDE’s KHTML by Apple for their Safari browser and later brought into Google’s Chrome, Samsung’s phone browsers and more.  
+
+“Few other browser engines have such portability and utility,” PulkoMandy said.  
+
+Haiku developer Ryan Leavengood started efforts to port the engine to create WebPositive in 2008. In 2009, WebPositive won a slot as part of the Google Summer of Code, and student developer Maxime Simon was able to get the browser in a “mostly usable” state while developer Stephan Aßmus (stippi) built the browser framework itself.   
+
+When PulkoMandy was hired, first as a paid developer before switching over as a volunteer to work on the WebKit port under WebPositive, there wasn’t a ton of new development going on with the software.  
+
+WebPositive wasn’t the only browser for Haiku, but with the additional portability and power of the WebKit1 engine as it existed at the time, the team decided to build it into  WebPositive and make it the official browser for the OS.
```
Comment (korli) [inline content/blog/animortis/2021-08-10_positively_critical_webpositive_and_haiku.md:25]:
```suggestion
WebPositive wasn’t the only browser for Haiku, but with the additional portability and power of the WebKit1 engine as it existed at the time, the team decided to build it into WebPositive and make it the official browser for the OS.
```

**Comment by pulkomandy** [comment]
> @pulkomandy should really review this one :-)

I have already reviewed it privately after being interviewed by the marketing team.

**Comment by aomgiwjc** [comment]
Hey guys, anything else I can do to help move this up?

**Comment by kallisti5** [comment]
I think we need to add some call to action seeking help on Webpositive.  Help getting our patches upstream, or help on our port

**Comment by aomgiwjc** [comment]
Added for the most part!

**Comment by jt15s** [comment]
Does this need any more work or can it be merged?

**Pull request merged**